packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+# Slider
+
+## Component Description
+
+The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr current-value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
+
+### Outputs
+
+None
+
+### Events
+
+- @change: Fires whenever the slider's value is changed
+
+### Slots
+
+- default: the primary slot intended for the `input` element
+- thumb: the draggable element of the slider
+- lower-thumb: same as `thumb`, but used in `range` mode
+- upper-thumb: same as `thumb`, but used in `range` mode
+- track: the slider's base and used to display the min/max selectable values
+- progress-track: the progress indicator when in `range` mode
+
+### CSS Variables
+
+None
+
+### Interactions
+
+#### thumb
+
+- On :hover a tooltip appears indicating the current value of the slider
```

Comment:
Will this be baked in? The implications of this are that slider now has a dependency on tooltip _in all scenarios_.

Reply:
I don't think we should bake it in. Sounds like you agree. Let's leave it out and I'll update the Spec to document the differences between this and React.